new-toast-site/package.json

```diff
@@ -23,6 +24,7 @@
     "facepaint": "^1.2.1",
     "fs-extra": "^9.0.1",
     "globby": "^11.0.1",
+    "json5": "^2.1.3",
```

Comment:
I use this in fetch-mdx-files to parse some JS objects out (because JSON.parse doesn't work for JS objects)